Problem1.cpp

```diff
@@ -0,0 +1,19 @@
+//Time Complexity - O(n^2)
```

Comment:
Try with unordered map Akshay. This one is not optimal and knowing the optimal solution for two sum is really important